jupyterhub/templates/image-puller/serviceaccount-continuous.yaml

```diff
@@ -0,0 +1,16 @@
+{{- /*
+ServiceAccount for the continuous hook's
+*/}}
+{{- if .Values.prePuller.hook.serviceAccount.create -}}
+{{- if .Values.prePuller.continuous.enabled -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: "continuous-image-puller-serviceaccount"
+  labels:
+    {{- include "jupyterhub.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "0"

```

Comment:
The hook annotations should be removed from here - this should be a normal reaource